seleniumtests/browse_selenium_test.py

```diff
@@ -0,0 +1,38 @@
+"""Tests for Browse page."""
+
+import unittest
+
+from selenium.webdriver.support.ui import WebDriverWait
```

Comment:
I am not too familiar with the documentation style for imports so if this is wrong lmk